<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPanel</class>
 <widget class="QMainWindow" name="ConfigPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigPanel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>179</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>OV Acquisition Server Connection</string>
    </property>
    <widget class="QLineEdit" name="addressOvAs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portOvAs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>15361</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectOvAsBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Speller Configuration</string>
    </property>
    <widget class="QLineEdit" name="stimulusDuration">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>26</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stimulus Duration</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="interTrialPause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ISI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>79</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Inter Trial Pause</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="interStimulusDuration">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberOfRepetition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of repetition</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberOfTrials">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Trials</string>
     </property>
    </widget>
    <widget class="QPushButton" name="initSpeller">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Init Speller</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="desiredPhrase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HELLO OKBA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Desired phrase</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Spelling mode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="spellingModeChoices">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Calibration</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Copy Mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Free Mode</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="spellerType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Flashing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Faces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>motion</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speller Type</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="startSpeller">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseSpeller">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopSpeller">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitSpeller">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Hybrid BCI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
